domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -56,6 +57,8 @@
     private final ClaimantResponse claimantResponse;
     private final LocalDateTime countyCourtJudgmentIssuedAt;
     private final LocalDate directionsQuestionnaireDeadline;
+    @DateNotInTheFuture
+    private final LocalDate moneyReceived;
```

Comment:
I suggest rename to moneyReceivedOn so that it sounds like pattern we have used for date types

Reply:
Will do